src/index.ts

```diff
@@ -135,11 +133,5 @@ export const run = async (config: KeeperConfig) => {
   }
 };
 
-logProcessError({
-  log(err, level) {
-    logger.log(level, `${err}, ${err.stack}`);
-  },
-});
```

Comment:
I found this useful, logs `uncaught exceptions` and `unhandledrejection` before crashing. But maybe your other metrics covers this?

Reply:
Yeah, I've got a few metrics to count errors but also a bunch of logging try/catch to prevent this. The only time where it might happen is operations that create promises where we don't `await`. So far the only area where we do that is when we treat metrics like a UDP request but I can see this being useful if/when bugs. I'll add it back.